samples/sync-write-1d-contig-contig-read-CS.json

```diff
@@ -0,0 +1,55 @@
+{
+    "mpi": {
+        "command": "mpirun",
+        "ranks": "1",
+        "configuration": "--allow-run-as-root -n 1 --oversubscribe"
```

Comment:
Is this specific to the environment where it was run? How would this change if we have to run on Perlmutter or some other system?

Reply:
This was taken from existing sample configuration files. I took the file `sync-write-1d-contig-contig-read-full.json` as reference for the new configs. This part would be changed just like for other existing patterns on a new system.

Reply:
Our CI should be able to handle that change when running in Perlmutter.